src/components/NavBar.tsx

```diff
@@ -26,11 +26,12 @@ const activeLink = {
 }
 const linkStyle = {
   display: 'inline-block',
-  margin: 2,
+  mx: 2,
   fontWeight: 700,
   letterSpacing: 0.8,
   textDecoration: 'none',
   color: '#dafffb',
+  borderBottom: '4px solid transparent',
```

Comment:
I don't know that the border should exist on the NavBar

I feel like the pages themselves should be responsible for adding margin on top, in case we want something to be like fluid with the NavBar (like some background for example)

Reply:
The borderBottom is for navbar item underline when page is active, and yeah each page is doing the margin top themselves.